app/src/main/java/com/example/bookwormadventuresdeluxe2/RequestDetailViewFragment.java

```diff
@@ -0,0 +1,167 @@
+package com.example.bookwormadventuresdeluxe2;
+
+import android.os.Bundle;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.example.bookwormadventuresdeluxe2.Utilities.DetailView;
+
+/**
+ * Holds the view for seeing details on a book in the Requested tab
+ * The user will be able to interact with request options on the book
+ */
+//TODO: add status specific buttons, functions, and labels
+public class RequestDetailViewFragment extends DetailView
+{
+    private Button btn1;
+    private Button btn2;
+    private TextView exchange;
+
+    public RequestDetailViewFragment()
+    {
+        // Required empty public constructor
+    }
+
+    public static RequestDetailViewFragment newInstance(String param1, String param2)
+    {
+        RequestDetailViewFragment fragment = new RequestDetailViewFragment();
+        return fragment;
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState)
+    {
+        this.bookDetailView = inflater.inflate(R.layout.fragment_request_detail_view, null, false);
+        ((TextView) bookDetailView.findViewById(R.id.app_header_title)).setText(R.string.requests_title);
+
+        // Setup back button
+        super.onCreateView(inflater, container, savedInstanceState);
+
+        this.btn1 = this.bookDetailView.findViewById(R.id.requestDetail_btn1);
+        this.btn2 = this.bookDetailView.findViewById(R.id.requestDetail_btn2);
+        this.exchange = this.bookDetailView.findViewById(R.id.request_exchange_location);
+
+        //TODO: do different stuff based on book status
+
```

Comment:
Again with the comments. If you need them keep them, I much prefer the smaller PR!

Reply:
Those segments are for changing the DetailView depending on the status. I worked on that first and commented it out when no one else had started working on the status yet. One thing at a time :)